src/index.js

```diff
@@ -3,6 +3,13 @@
 'use strict';
 
 const { createServer } = require('./createServer');
+const { Expense } = require('./models/Expense.model');
+const { User } = require('./models/User.model');
+
+(async function syncDb() {
+  await User.sync({ force: true });
+  await Expense.sync({ force: true });
+})();
```

Comment:
Add error handling to asynchronous operations to catch any potential errors. Also it would be good to just console.log if db has succesfully synced. You could also sync whole `db` by calling `sync` method on `sequlize` instance.